### PR TITLE
[CC-14556] AC Doc Documentation Discount valid from/to supports time definition

### PR DESCRIPTION
**Pull request by VadymSachenko**
documenting the feature
Proofreading the documents

## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ The following table describes the attributes you enter and select in the *Genera
...
 | NON-EXCLUSIVE | Defines the discount exclusivity. Buyers can redeem non-exclusive discounts in conjunction with other non-exclusive discounts.|
 | EXCLUSIVE | Defines the discount exclusivity. When a discount is exclusive, no other discounts may be applied in conjunction. When a cart is eligible for multiple exclusive discounts, the discount with the highest value to the customer is applied. The exception to this is promotional product discounts. Query string discounts and promotional product discounts exclude only among each other. Promotional product discounts are not affected by exclusive query string discounts, and vice versa.|
-| VALID FROM and VALID TO| Vouchers are redeemable/the cart rule is active between Valid From and Valid To dates, inclusive. E.g., a voucher can be redeemed/discount applies to the cart starting from 1/1/2018 until 31/12/2019.|
+| VALID FROM (TIME IN UTC) and VALID TO (TIME IN UTC)| Vouchers are redeemable or the cart rule is active between **VALID FROM (TIME IN UTC)** and **VALID TO (TIME IN UTC)** dates, inclusive. For example, a voucher can be redeemed or discount applies to the cart starting from 01.12.2021 00:00 until 31.01.2022 23:59 by UTC|
```
Comment (JacquieRooney) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:128]:
```suggestion
| VALID FROM and VALID TO | Vouchers are redeemable or the cart rule is active between **VALID FROM** and **VALID TO** dates and times (in UTC), inclusive. For example, a voucher can be redeemed or discount applies to the cart starting from 01.12.2021 23:00 until 31.01.2022 22:59, UTC |
```

Comment (JacquieRooney) [reply docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:128]: I don't think it's necessary to state (TIME IN UTC) next to the fields. It's enough to add "dates and times" and specify non-conventional valid from/to times in the example to get our point across.

**Review by JacquieRooney — CHANGES_REQUESTED** [review]

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md (hunk 1 above)
Comment (VadymSachenko) [reply docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md]: Changes accepted. Thanks.

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ The following table describes the attributes you enter and select in the *Genera
...
 | NON-EXCLUSIVE | Defines the discount exclusivity. Buyers can redeem non-exclusive discounts in conjunction with other non-exclusive discounts.|
 | EXCLUSIVE | Defines the discount exclusivity. When a discount is exclusive, no other discounts may be applied in conjunction. When a cart is eligible for multiple exclusive discounts, the discount with the highest value to the customer is applied. The exception to this is promotional product discounts. Query string discounts and promotional product discounts exclude only among each other. Promotional product discounts are not affected by exclusive query string discounts, and vice versa.|
-| VALID FROM and VALID TO| Vouchers are redeemable/the cart rule is active between Valid From and Valid To dates, inclusive. E.g., a voucher can be redeemed/discount applies to the cart starting from 1/1/2018 until 31/12/2019.|
+| VALID FROM and VALID TO | Vouchers are redeemable or the cart rule is active between **VALID FROM** and **VALID TO** dates and times (in UTC), inclusive. For example, a voucher can be redeemed or discount applies to the cart starting from 01.12.2021 23:00 until 31.01.2022 22:59, UTC |
```
Comment (lenadoc) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:128]:
```suggestion
| VALID FROM and VALID TO | Vouchers are redeemable or the cart rule is active between **VALID FROM** and **VALID TO** dates and times (in UTC), inclusive. For example, a voucher can be redeemed or discount applies to the cart starting from 01.12.2021 23:00 until 31.01.2022 22:59, UTC. |
```

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md [hunk 3]
```diff
@@ -330,4 +330,4 @@ This section contains a set of tables that describe fields, value types, and ope
 |Greater or equal|>=|Number|Checks if the value is greater than or equal to the value of the right operand|
 
 **What's next?**
-<br>See [Managing Discounts](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/managing-discounts.html) to know more about the actions you can do once the discount is created.
\ No newline at end of file
+<br>See [Managing Discounts](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/managing-discounts.html) to know more about the actions you can do once the discount is created.
```
Comment (lenadoc) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:333]:
```suggestion
<br>See [Managing Discounts](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/managing-discounts.html) to learn more about the actions you can do once the discount is created.
```

**Review by lenadoc — DISMISSED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**